config/prow/config_knative.yaml

```diff
@@ -504,6 +504,36 @@ presubmits:
     - --run-test
     - ./test/reconciler-tests.sh
   - go-coverage: true
+  - custom-test: channel-integration-tests-tls
+    needs-dind: true
+    args:
+    - --run-test
+    - ./test/e2e-tests.sh --channel-tls
+  - custom-test: channel-integration-tests-sasl-ssl
+    needs-dind: true
+    args:
+    - --run-test
+    - ./test/e2e-tests.sh --channel-sasl-ssl
+  - custom-test: channel-integration-tests-sasl-plain
+    needs-dind: true
+    args:
+    - --run-test
+    - ./test/e2e-tests.sh --channel-sasl-plain
+  - custom-test: channel-reconciler-tests-tls
+    needs-dind: true
+    args:
+    - --run-test
+    - ./test/reconciler-tests.sh channel-tls
+  - custom-test: channel-reconciler-tests-sasl-ssl
+    needs-dind: true
+    args:
+    - --run-test
+    - ./test/reconciler-tests.sh --channel-sasl-ssl
+  - custom-test: channel-reconciler-tests-sasl-plain
+    needs-dind: true
+    args:
+    - --run-test
+    - ./test/reconciler-tests.sh --channel-sasl-plain
```

Comment:
But, the knative-sandbox/eventing-kafka-broker#1816 isn't merged. When we merge https://github.com/knative/test-infra/pull/3062 first, it's likely it will block other PRs!

Better, would be to spit knative-sandbox/eventing-kafka-broker#1816 in two. 

1. First introduce, new flags, and do NOP to make sure the test will not fail. 
2. Then merge https://github.com/knative/test-infra/pull/3062. 
3. Lastly, merge the knative-sandbox/eventing-kafka-broker#1816, so you will have a clear signal.

Reply:
@cardil I explained pretty much the same thing in this PR's description at the top.

New flags don't have an effect. So, everything is already like you explained in your comment above.

Reply:
Hope you're right about it.